### DISCUSSION TITLE
Certain feeds return no entries

### DISCUSSION THREAD

**Discussion by prescientmoon** [Athou/commafeed#1305] [General]
**Describe the bug**
For me, the feed at https://faultlore.com/blah/rss returns no entires on the commafeed website. Running `curl https://faultlore.com/blah/rss` from the server the webapp is hosted on returns all the entries. I have already tried using a randomly generated useragent like the one below, but that doesn't change anyhing: 
```
CF_APP_USERAGENT = "Mozilla/5.0 (iPhone; CPU iPhone OS 11_0_6; like Mac OS X) AppleWebKit/533.48 (KHTML, like Gecko)  Chrome/49.0.2557.162 Mobile Safari/602.0"
```

The commafeed webapp returns `'304 - not modified' http code received`. I have additionally tried to add a new feed with the url `https://faultlore.com/blah/rss?something` to try to perhaps go around whatever caching commafeed might perform, but nothing changed.

**Screenshots**
![image](https://github.com/Athou/commafeed/assets/39400800/49c1d0f6-bf84-4e5e-984d-a019af96f485)

**Environment (please complete the following information):**

- CommaFeed version 4.3.1 (fafd4c9).
- Browser & device: tried on mobile firefox, mobile chrome, and firefox on linux (nixos)

**Comment by prescientmoon** [comment]
Hmmm, I think the issue might be that no entries are newer than the default 365 days (I did not notice this configuration variable)

**Comment by prescientmoon** [comment]
Nope, it looks like the following doesn't change anything
```
CF_APP_MAXENTRIESAGEDAYS = "0"
```

**Comment by prescientmoon** [comment]
Nevermind, it looks like setting that env var won't affect older feeds unless you delete, and add them again with a query param at the end of the url (i.e. `https://gankra.github.io/blah/rss?something`). This feels pretty counterintuitive, but I got it to work at least...

**Comment by Athou** [comment]
You're right, CommaFeed does not insert entries older than `maxEntriesAgeDays` in the database because they would be deleted during the next cleanup. You're also right that changing the value of `maxEntriesAgeDays` does not invalidate the cached values of the feed.

> Nevermind, it looks like setting that env var won't affect older feeds unless you delete, and add them again with a query param at the end of the url (i.e. https://gankra.github.io/blah/rss?something). 

Feeds with no subscriptions are deleted every hour during the cleanup task, that's why when you unsubscribe then immediately resubscribe to a feed, cache values are still present and no entries are added.

**Comment by nleanba** [comment]
Is there a way to force reload a feed without having to unsubscribe?

I naively updated my commafeed instance and it apparently deleted (without warning) 3K entries it deemed "too old", at least 800 of which were unread.

1. It seems bad to me that such a setting which may cause quite some data loss is enabled by default if you update the docker container
2. Is there any way at all to get these entries back? Ideally with their read/unread status, but at the very least I'd like to have them without having to manually resubscribe to all my feeds
